ui/src/components/head/index.js

```diff
@@ -5,11 +5,31 @@
 // - https://css-tricks.com/inheriting-box-sizing-probably-slightly-better-best-practice
 
 import loadApp from './__inline__/loadApp';
+import loadFonts from './__inline__/loadFonts';
 import resetCSS from './__inline__/reset.css';
-import fontsCSS from './__inline__/fonts.css';
 
 //  Having to typecast loadApp to a string here to appease flow
 const loadAppStr: string = (loadApp: any);
+const loadFontsStr: string = (loadFonts: any);
+
+const getFontDefinitions = (fontDefinitions: Array<Object>): string => {
```

Comment:
Can we define types for `fontDefinitions`, `typeFace`, `fileType` and `hintType`?